### ISSUE TITLE
ctpfrec outputs training error, unlike hpfrec which outputs validation error

### ISSUE THREAD

**Issue by JackMack21** [david-cortes/ctpfrec#4]
I've noticed when training the ctpfrec model it outputs training error, unlike hpfrec which outputs validation error. This is going to be problematic for model selection as obviously the training error will continue to decrease with model complexity and therefore result in overfitting on the test set. Do you have any advice on how I can find out the validation error?

Thank you

**Comment by david-cortes** [comment]
Use parameter `stop_crit="val-llk"`. Then it will show you the validation error.
